datalad/cmdline/helpers.py

```diff
@@ -255,7 +255,6 @@ def get_repo_instance(path=curdir, class_=None):
     else:
         raise RuntimeError("No datalad repository found in %s" % abspath_)
 
-
```

Comment:
Unrelated change ideally to be reverted (two empty lines too separate at this level I believe)

Reply:
Fixed locally.